config/queues.yaml

```diff
@@ -1,5 +1,8 @@
 partitions:
   - name: default
+    globalpolicies:
+      - name: myschedulingpolicy
+        policy: binpacking
```

Comment:
Should we make the global_policy to be more extendable? I'm wondering why we need a name field here? Should it be just a list of policy with per-policy parameters?

Reply:
And how abt call it "node_sort_policy" to make it to be more specific?

Reply:
The arbitrary name is quite confusing to me.

Reply:
i see, i was more less thinking that we can choose a custom policy injection later, so a name will be helpful for user to define any polices per partition, global or queue level. so they can refer the policy once defined at the top level, just referring name alone later.

node_sort_policy can be of any sort. we can sort by descending , or packing model,  may be looking at some different score per node etc. 

so do u think it should be like
      - policy: node_sort_policy
           type: roundrobin/binpacking
?